README.md

```diff
@@ -149,7 +149,7 @@ JavaScript: SDK|Every Wednesday at 09:00 PT|[Google Doc](https://docs.google.com
 PHP: SDK|Every Wednesday at 10:30 PT|[Google Doc](https://docs.google.com/document/d/1WLDZGLY24rk5fRudjdQAcx_u81ZQWCF3zxiNT-sz7DI/edit)|[Slack](https://cloud-native.slack.com/archives/C01NFPCV44V)|
 Python: SDK|Every Thursday at 09:00 PT|[Google Doc](https://docs.google.com/document/d/1CIMGoIOZ-c3-igzbd6_Pnxx1SjAkjwqoYSUWxPY8XIs/edit)|[Slack](https://cloud-native.slack.com/archives/C01PD4HUVBL)|
 Ruby: SDK|Every Tuesday at 09:00 PT|[Google Doc](https://docs.google.com/document/d/1D15bO8o340sQm2CVZiukEJuCO_XMMHKPuTznoEhyFqE/edit)|[Slack](https://cloud-native.slack.com/archives/C01NWKKMKMY) and [GitHub Discussions](https://github.com/open-telemetry/opentelemetry-ruby/discussions)|
-Rust: SDK|N/A|[Google Doc](https://docs.google.com/document/d/1tGKuCsSnyT2McDncVJrMgg74_z8V06riWZa0Sr79I_4)|[Gitter](https://gitter.im/open-telemetry/opentelemetry-rust)|
+Rust: SDK|Every Tuesday at 16:00 UTC|[Google Doc](https://docs.google.com/document/d/1tGKuCsSnyT2McDncVJrMgg74_z8V06riWZa0Sr79I_4)|[Slack](https://cloud-native.slack.com/archives/C03GDP0H023) and [Gitter](https://gitter.im/open-telemetry/opentelemetry-rust)|
```

Comment:
the other meetings are specified in Pacific Time, is there a specific desire for this to be in UTC?

Reply:
also, I'd recommend dropping Gitter, the other SIGs have switched to Slack and dropped Gitter

Reply:
UTC is what all timezone offsets are based from, and so we've chosen that. 
In the rust SIG the slack and the gitter are valid. Slack is the "official" but gitter is still used by some of the maintainers who have no desire to use Slack.

Reply:
> Slack is the "official" but gitter is still used by users who have no desire to use Slack.

sure, none of the other SIGs are still using gitter, but I think it's ok if the Rust SIG explicitly prefers to continue using it

> Honestly I was surprised that the other sigs had pacific time at all

I think generally the meetings are specified in US time so that daylight savings changes will affect the least number of people

it's ok if the Rust SIG prefers otherwise, and I can update the meeting on the public calendar to be explicitly in UTC, just let me know

> No-one knows how that relates to them outside of the US.

we're working on fixing the [calendar sync'ing](https://github.com/open-telemetry/community/pull/1415) so that people can subscribe more easily and get the meeting invites in their local time

Reply:
Yes please if you can make it UTC in the calendar.